API/RestSharpAPI/Hooks/RestSharpHooks.cs

```diff
@@ -1,25 +1,109 @@
-﻿using RestSharp;
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Gherkin.Model;
+using AventStack.ExtentReports.Reporter;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using NLog;
+using NLog.Web;
+using NUnit.Framework;
+using RestSharp;
+using System;
+using System.IO;
+using System.Text;
 using TechTalk.SpecFlow;
-using TestFramework.Hooks;
+using TechTalk.SpecFlow.Infrastructure;
+
+using Utilities;
 
 namespace RestSharpApi.Hooks
 {
     [Binding]
-    public sealed class RestSharpHooks 
+    public  class RestSharpHooks 
     {
         public static RestClient _restClient;
+        public static string ProjectPath = Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName;
+        public static string PathReport = ProjectPath + "\\TestResults\\Report\\ExtentReport.html";
```

Comment:
There is a PathReport variable defined here, again we're creating a new folder at line no 44

Reply:
fixed